packages/nodejs/src/client.ts

```diff
@@ -73,7 +71,14 @@ export class BaseClient implements Client {
 
     this.instrumentation = new Instrumentation(this.tracer(), this.metrics())
 
-    initCorePlugins(this.instrumentation, { ignoreInstrumentation })
+    const { instrumentRedis, instrumentHttp, instrumentPg } = this.config.data
+    initCorePlugins(this.instrumentation, {
+      instrumentationConfig: {
+        http: instrumentHttp,
```

Comment:
```suggestion
        http: instrumentHttp,
        https: instrumentHttp,
```

The value of the `instrumentHttp` configuration option should be passed in this `instrumentationConfig` as the value for both the `http` and `https` keys. Otherwise, the HTTPS integration will never be disabled.